### PR TITLE
Confiscates the bounty hunter flamethrower, gives them a fire cycler shotgun + fire pistol

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Some lovely person who will go unnamed decided 3 years ago that the flamethrowers would be a good weapon for some bounty hunters to use to hunt their quarry. This turned out to be a massive liability and cause lots of innocent people to get toasted, so this PR aims to give them a different fire-based arsenal that isn't so indiscriminate.

The armored bounty hunter role is now issued a cycler dual-tube shotgun that comes with one tube of rubbershot, and one tube of a new incendiary shotgun slug that leaves no fire trail, but still lights up their target. They also receive a 9mm Makarov loaded with fire bullets and one spare mag (these ones do leave a trail).

## Why It's Good For The Game
When an armored bounty hunter unloaded a flamethrower on the emergency shuttle trying to catch a fugitive, I thought for sure he was lying when he said that was the only weapon he got. Turns out he was right! Neither of the guns are as powerful as a flamethrower, but hopefully together they can be a good substitute.

## Changelog

:cl: Ryll/Shaps
balance: Armored bounty hunters are no longer being issued flamethrowers. Instead, they are now issued dual-tube shotguns filled with rubbershot and trailless incendiary slugs, as well as a 9mm Makarov loaded with fire bullets.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
good!

**Comment by Cheshify** [comment]
I was literally going to go in and rework all these loadouts, this is incredibly cool.

File: code/modules/projectiles/projectile/bullets/_incendiary.dm [hunk 1]
```diff
@@ -11,10 +14,11 @@
 
 /obj/projectile/bullet/incendiary/Move()
 	. = ..()
-	var/turf/location = get_turf(src)
-	if(location)
-		new /obj/effect/hotspot(location)
-		location.hotspot_expose(700, 50, 1)
+	if(leaves_fire_trail)
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/_incendiary.dm:17]: early returns?

**Comment by Fikou** [comment]
shotguns are already the gimmick of the hook bounty hunter, this feels like its making the armored one superior to the other ones which is kinda lame

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 2]
```diff
@@ -72,20 +72,29 @@
...
 
 /obj/item/gun/ballistic/shotgun/automatic/dual_tube/examine(mob/user)
 	. = ..()
 	. += span_notice("Alt-click to pump it.")
 
 /obj/item/gun/ballistic/shotgun/automatic/dual_tube/Initialize(mapload)
 	. = ..()
-	if (!alternate_magazine)
-		alternate_magazine = new mag_type(src)
+	alt_mag_type = alt_mag_type || mag_type
```
Comment (MrMelbert) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:96]:
```suggestion
	alt_mag_type ||= mag_type
```
if you wanna used a cursed operator

File: code/modules/projectiles/projectile/bullets/_incendiary.dm [hunk 3]
```diff
@@ -11,10 +14,12 @@
...
 	. = ..()
+	if(!location || !leaves_fire_trail)
+		return
+
 	var/turf/location = get_turf(src)
-	if(location)
-		new /obj/effect/hotspot(location)
-		location.hotspot_expose(700, 50, 1)
+	new /obj/effect/hotspot(location)
+	location.hotspot_expose(700, 50, 1)
```
Comment (MrMelbert) [inline code/modules/projectiles/projectile/bullets/_incendiary.dm:22]: `if(location)` has to be moved back

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/fugitive/fugitive_outfits.dm [hunk 4]
```diff
@@ -159,15 +159,21 @@
...
+	l_pocket = /obj/item/ammo_box/magazine/m9mm/fire
 	id = /obj/item/card/id/advanced/bountyhunter
-	l_hand = /obj/item/tank/internals/plasma/full
-	r_hand = /obj/item/flamethrower/full/tank
+	l_hand = /obj/item/gun/ballistic/shotgun/automatic/dual_tube/bounty
+
+	backpack_contents = list(
+		/obj/item/ammo_casing/shotgun/rubbershot = 4,
+		/obj/item/ammo_casing/shotgun/incendiary/no_trail = 4,
+		)
```
Comment (tralezab) [inline code/modules/antagonists/fugitive/fugitive_outfits.dm:176]: sorry to be a stickler but unindent this `)`

**Review by tralezab — CHANGES_REQUESTED** [review]
JUST SUPER QUICK

**Pull request merged**